neutrons/histogram-dev
# Tests

# Short description of the changes:

Set up pytest, surpressed and fixed all tests warnings. Now the tests can be run in root directory through command pytest. All temporary files created by tests are in tmp folder managed by tempfile python package. The testing data is in tests/data folder.
[Story 6592](https://ornlrse.clm.ibmcloud.com/ccm/web/projects/Neutron%20Data%20Project%20(Change%20Management)#action=com.ibm.team.workitem.viewWorkItem&id=6592)
# Long description of the changes:


# Check list for the pull request

# Check list for the reviewer

# Manual test for the reviewer

Set up environment using environment.yml. Go into python folder install with `pip install -e.`
Pull the tests branch,  in root directory type `pytest`, 145 tests should pass, no additional files are created, no warnings should pop up.
# References
